recipes/opencolorio/all/conanfile.py

```diff
@@ -50,7 +53,11 @@ def requirements(self):
         # TODO: add GLUT (needed for ociodisplay tool)
         self.requires("lcms/2.12")
         self.requires("yaml-cpp/0.7.0")
-        self.requires("tinyxml/2.6.2")
+        if tools.Version(self.version) < "2.1.0":
+            self.requires("tinyxml/2.6.2")
+        self.requires("pystring/1.1.3")
```

Comment:
Are they compatible with version 1.x ?

Reply:
OCIO 1.1.1 bundles pystring 1.1.2. requires tinyxml 2.6.2
OCIO 2.1.0 downloads pystring 1.1.3. does not require tinyxml

so I'll just require pystring for 2.x